v11.13.1 has bug on linux platform, resolve failed

**oxc-resolver@11.13.1  on ubuntu  resolve failed ** 

 oxc-resolver@11.13.0 work successful


```js
import { ResolverFactory } from 'oxc-resolver';

const resolver = new ResolverFactory({});
const directory = process.cwd();
const specifier = './package.json';
const resolution = resolver.sync(directory, specifier);

console.log('directory', directory);
console.log('specifier', specifier);
console.log('resolution', resolution);
```

output

```bash
node resolve.mjs
directory /home/coder/cbuild-demo
specifier ./package.json
resolution { error: "Cannot find module './package.json'" }
```

--- a/src/file_system.rs
+++ b/src/file_system.rs
@@ -185,12 +185,7 @@ impl FileSystemOs {
                 Ok(crate::windows::symlink_metadata(path)?.into())
             } else if #[cfg(target_os = "linux")] {
                 use rustix::fs::{AtFlags, CWD, FileType, StatxFlags};
-                let statx = rustix::fs::statx(
-                    CWD,
-                    path,
-                    AtFlags::STATX_DONT_SYNC | AtFlags::SYMLINK_NOFOLLOW,
-                    StatxFlags::TYPE,
-                )?;
+                let statx = rustix::fs::statx(CWD, path, AtFlags::SYMLINK_NOFOLLOW, StatxFlags::TYPE)?;
                 let file_type = FileType::from_raw_mode(statx.stx_mode.into());
                 Ok(FileMetadata::new(file_type.is_file(), file_type.is_dir(), file_type.is_symlink()))
             } else {

